pkg/cli/admin/release/mirror.go

```diff
@@ -885,15 +938,30 @@ func printImageContentInstructions(out io.Writer, from, to string, signatureToDi
 	}
 	delete(unstructuredObj.Object["metadata"].(map[string]interface{}), "creationTimestamp")
 
-	icspExample, err := yaml.Marshal(unstructuredObj.Object)
+	icspDataBytes, err := yaml.Marshal(unstructuredObj.Object)
 	if err != nil {
-		return fmt.Errorf("Unable to marshal ImageContentSourcePolicy example yaml: %v", err)
+		return fmt.Errorf("Unable to marshal ImageContentSourcePolicy yaml: %v", err)
+	}
+	icspFile := o.ICSPTo
```

Comment:
grrr ok

Reply:
The release-signature configmap change that writes the configmap to disk added a flag, 
`release-image-signature-to-dir` that I modified in this PR to `release-image-config-dir` so it can be dual purpose.  This requires a change to current users, though, so we might want to keep the release-image-signature-to-dir and add a new flag for the icsp-to-dir.

Reply:
updated to add a parallel flag --release-image-icsp-to-dir  instead, after our discussion.